dev/setup-multi-cluster

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+if [ ! -f dev/envrc-mc ]; then
+    echo "Run this from the root of the repo"
+    exit 1
+fi
+
+./dev/k3d-clean
+
+if [ -n "$ENVRC" ]; then
```

Comment:
I see the convenience, but I'm not sure if the name is right. If we treat this like config, maybe we should call it config. 

What do you think about?
* .env.multicluster  (or something) 
* .env.singlecluster
* dev/env.multicluster-defaults
* dev/env.singlecluster-defaults

What trips me up is that `.envrc` is managed by https://direnv.net/. It's used to provide default env vars for different directories. 
I think `.env` files are more commonly loaded by the script/app itself (dotenv), not by the shell. That's also why we'd want to avoid `.env` without a suffix, it's loaded by Docker for example.

Reply:
As shortly discussed, I'll change the environment variable to `FLEET_TEST_CONFIG`. I also very much appreciate the other suggestions, of which I like a slight adaptation the most:

- dev/env.multi-cluster-defaults
- dev/env.single-cluster-defaults

Reply:
Due to the greater proximity to the names of the scripts, `dev/setup-single-cluster` and `dev/setup-multi-cluster`.